CHANGELOG.md

```diff
@@ -4,15 +4,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Upcoming version 5.x.x (`develop` branch)
 
-[#709]
```

Comment:
Seems like a merge conflict? I think this change should stay here.

Reply:
Fixed now!